spec/datetimeformat.html

```diff
@@ -592,7 +600,7 @@
       </p>
 
       <p>
-        The function returns a new object whose properties and attributes are set as if constructed by an object literal assigning to each of the following properties the value of the corresponding internal slot of this DateTimeFormat object (see <emu-xref href="#sec-properties-of-intl-datetimeformat-instances"></emu-xref>): locale, calendar, numberingSystem, timeZone, hour12, weekday, era, year, month, day, hour, minute, second, and timeZoneName. Properties whose corresponding internal slots have the value *undefined* are not assigned.
+        The function returns a new object whose properties and attributes are set as if constructed by an object literal assigning to each of the following properties the value of the corresponding internal slot of this DateTimeFormat object (see <emu-xref href="#sec-properties-of-intl-datetimeformat-instances"></emu-xref>): locale, calendar, numberingSystem, timeZone, hourCycle, weekday, era, year, month, day, hour, minute, second, and timeZoneName. Properties whose corresponding internal slots have the value *undefined* are not assigned.
```

Comment:
If anyone is using the `hour12` field of the resolvedOptions(), this patch makes it go away. That risks web compatibility. For that reason, I'd rather we not merge this patch.

One way to resolve this would be to hold around the `hour12` option that the users passed in, and then just pass that back out. I'd suggest, on the other hand, to just use the \[\[HourCycle]] internal slot that we already have: if the hourCycle is one of the two values that hour12 would provide, then use that in an `hour12` property; otherwise, omit the `hour12` property.

In my opinion, it's OK to have `hour12` in the output even if it wasn't in the input. This is just another instance of the pattern where `resolvedOptions` has options that the user didn't provide, but were chosen by, e.g., locale selection or matching a pattern.

I've made this suggestion a couple times in the past; I must not have been explaining myself clearly. I'll follow up with spec text to explain the change.